compiler+runtime/src/cpp/jank/runtime/obj/repeat.cpp

```diff
@@ -16,6 +16,7 @@ namespace jank::runtime::obj
     : value{ value }
     , count{ count }
   {
+    assert(0 < to_int(count));
```

Comment:
Feels like this could be an exception, rather than assertion, since someone embedding jank may create it a repeat with `0` and we don't need to kill the whole program.

Reply:
Done.

It doesn't necessarily need to be an error either. We can made these constructors robust by adding checks in its members.